index.js

```diff
@@ -214,23 +214,24 @@ SteamTradeOffers.prototype.cancelOffer = function(options, callback) {
   });
 };
 
-SteamTradeOffers.prototype.acceptOffer = function(options, callback) {
+SteamTradeOffers.prototype.acceptOffer = function(offer, callback) {
   var cb = function () {
     if (typeof callback === 'function') {
       callback.apply(null, arguments);
     }
   };
 
   this._requestCommunity.post({
-    uri: communityURL + '/tradeoffer/' + options.tradeOfferId + '/accept',
+    uri: communityURL + '/tradeoffer/' + offer.tradeofferid + '/accept',
     headers: {
-      referer: communityURL + '/tradeoffer/' + options.tradeOfferId + '/'
+      referer: communityURL + '/tradeoffer/' + offer.tradeofferid + '/'
     },
     json: true,
     form: {
       sessionid: this.sessionID,
       serverid: 1,
-      tradeofferid: options.tradeOfferId
+      tradeofferid: offer.tradeofferid,
+      partner: offer.steamid_other
```

Comment:
So basically adding `partner` as options param like `partnerSteamId` is all you had to do.